src/EncompassRest/Loans/Enums/DaysInYear.cs

```diff
@@ -0,0 +1,14 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum DaysInYear
+    {
+        [EnumMember(Value = "360")]
```

Comment:
match numeric values with string values (eg. n360 = 360)

Reply:
In case you didn't see I've done as you suggested.